### ISSUE TITLE
INSERT error: java.lang.NoSuchMethodError: io.milvus.grpc.RowRecord.emptyFloatList()

### ISSUE THREAD

**Issue by sealedtx** [milvus-io/milvus-sdk-java#132]
Hi! I have an error while inserting vectors. Using latest version of milvus (**0.10.3**) and latest version of java sdk (**0.8.5**).
Here is code snippet taken from examples:
```
        List<List<Float>> allEmbeddings = new ArrayList<>();
        allEmbeddings.add(...); # add vector 512 length

        InsertParam insertParam = new InsertParam.Builder(collectionName)
                .withFloatVectors(allEmbeddings)
                .build();
        InsertResponse insertResponse = milvusClient.insert(insertParam);
```

It produces an error:
```
Exception in thread "main" java.lang.NoSuchMethodError: 'com.google.protobuf.Internal$FloatList io.milvus.grpc.RowRecord.emptyFloatList()'
	at io.milvus.grpc.RowRecord.<init>(RowRecord.java:24)
	at io.milvus.grpc.RowRecord.<clinit>(RowRecord.java:694)
	at io.milvus.client.AbstractMilvusGrpcClient.buildRowRecordList(MilvusGrpcClient.java:1329)
	at io.milvus.client.AbstractMilvusGrpcClient.insert(MilvusGrpcClient.java:528)
	at io.milvus.client.MilvusGrpcClient.insert(MilvusGrpcClient.java:49)
	at com.incoresoft.Test.main(Test.java:78)
```

What is the problem? Anything wrong with protobuf generated models? Because I look on decompiled code of `RowRecord` and indeed, there is no `emptyFloatList()` method. 
Thank you.

**UPD**:
I tried same code in new project with zero dependecies - and it works. So the problem - incompatible dependencies. How can I figure out what exactly is wrong?

**Comment by sahuang** [comment]
I am unsure why that happens, because `RowRecord` is something related to grpc proto and is auto-generated upon compiling and installing all dependencies. One possibility is that there is a version update or something wrong happened during dependency installation, which may lead to incompatible issues.

Usually in this case, if you are in IDEs such as intellij, just maven clean, remove all dependencies and re-install, the problem will be gone.

**Comment by sealedtx** [comment]
@sahuang thank you for a quick repsonse. Cleaning and rebuilding didn't solve the problem. 
But I found a solution. Just in case someone faced with same problem. I looked through `mvn dependency:tree` and found that there are different version of com.google.protobuf
`com.google.protobuf:protobuf-java-util:jar:3.11.0` (used by milvus)
`com.google.protobuf:protobuf-java:jar:3.6.1` (used by other dep)

**Solution**:
I added exclusion
```
<exclusions>
 <exclusion>
  <groupId>com.google.protobuf</groupId>
  <artifactId>protobuf-java</artifactId>
 </exclusion>
</exclusions>
```
to other dep.
And imported separately this artifact but with same version as milvus uses:
```
<dependency>
 <groupId>com.google.protobuf</groupId>
 <artifactId>protobuf-java</artifactId>
 <version>3.11.0</version>
</dependency>
```
